model/accounts_mgmt/v1/feature_toggle_query_resource.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages feature toggle query.
+resource FeatureToggleQuery {
+	// Retrieves the details of the feature toggle by providing query context
+	method Post {
+		in Request FeatureToggleQueryRequest
```

Comment:
Ah okay, I see you're "posting" a query request and getting back a toggle setting? If `FeatureToggleQueryRequest` only has an Org ID, why can't we `GET /api/.../{org_id}/feature_toggle/{feature_toggle_id}`, like we do for all other resources?

Reply:
Feature toggle can have many parameters as input.
For example, we can decide to toggle feature by cluster, by region, or anything else.
Right now our only use case is for org.

Reply:
interesting... AMS has an interesting way of dealing with these arbitrary data types. They usually have a `--fetchFooBarData` param in their GET requests. Is this something we can do? For example `GET /../orgs/{org_id} --fetchFeatureToggles`. WDYT?